packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -1794,20 +1797,24 @@ class Flex extends MultiChildRenderObjectWidget {
   /// How the children should be placed along the main axis.
   final MainAxisAlignment mainAxisAlignment;
 
+  /// The limit used to compute free space along the main axis.
```

Comment:
please add a paragraph of additional material here explaining how this works and how you would use the various values and what effects the various values have